config/routes.rb

```diff
@@ -75,8 +75,9 @@
       resource :sessions
       resources :site_statuses, only: :update
 
-      resources :access_requests, only: :update do
+      resources :access_requests, only: %i[update show] do
```

Comment:
`show is not part of this PR so no need for that here but I would add `index` then remove the line you added below

Reply:
Done. Thanks.